nservicebus/upgrades/all-versions.include.md

```diff
@@ -0,0 +1,2095 @@
+### NServiceBus
+
+<table>
+  <tr>
+    <th>Version</th>
+    <th>Released</th>
+    <th>Supported until</th>
```

Comment:
I did consider it, but it would mean either padding all the columns to the width of the largest item anywhere in the table:

```markdown
| Version | Released   | Supported until | Explanation                   |
|---------|------------|-----------------|-------------------------------|
| 7.2.x   | 2017-03-24 | -               | -                             |
| 7.0.x   | 2016-10-11 | 2017-06-24      | Superseded by 7.2.x           |
| 6.2.x   | 2015-02-13 | 2018-10-11      | Extended by NServiceBus 5.2.x |
```

or not padding at all:

```markdown
| Version | Released | Supported until | Explanation |
|-|-|-|-|
| 7.2.x | 2017-03-24 | - | - |
| 7.0.x | 2016-10-11 | 2017-06-24 | Superseded by 7.2.x |
| 6.2.x | 2015-02-13 | 2018-10-11 | Extended by NServiceBus 5.2.x |
```

My appetite for both options was limited.

@SimonCropp how strongly do you feel about it?